test/test_connection.py

```diff
@@ -153,6 +150,14 @@ def test_ping_pong(self, as_client):
             data = wire_data[2:]
         assert data == payload
 
+        # Verify that connection vends the PongReceive event with the correct payload
```

Comment:
...I don't know what "vend" means in this context, so I just guessed you meant send :-). Maybe "yields"?